content/F2015/F15-event-code-party.md

```diff
@@ -1,21 +1,21 @@
-title: Code Party 
```

Comment:
I think you edited the wrong file - the file is definitely included in the commit because it was changed